common/src/main/java/jp/co/soramitsu/common/data/network/subquery/SumRewardResponse.kt

```diff
@@ -0,0 +1,5 @@
+package jp.co.soramitsu.common.data.network.subquery
+
+class SumRewardResponse(val sumReward: SumReward) {
+    class SumReward(val accountTotal: String)
```

Comment:
Why not just use BigInteger directly here? It is not possible since String can not be auto-converted by Gson to BigInteger?